client/keys/show.go

```diff
@@ -18,30 +22,67 @@ const (
 	FlagPublicKey = "pubkey"
 	// FlagBechPrefix defines a desired Bech32 prefix encoding for a key.
 	FlagBechPrefix = "bech"
+
+	flagMultiSigThreshold = "multisig-threshold"
```

Comment:
Is there a case where we'd want this exposed?

Reply:
I there is presently no case for it to be exposed.